app/utils/rijksregisternummer.js

```diff
@@ -0,0 +1,136 @@
+/* MIT License - Copyright © 2022 Dieter Luypaert <dieterluypaert@gmail.com> from https://github.com/moeriki/be-nrn
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the “Software”), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+const BIS_MONTH_INCREMENT_GENDER_UNKNOWN = 20;
+const BIS_MONTH_INCREMENT_GENDER_KNOWN = 40;
+
+const mod97 = (input) => String(97 - (Number(input) % 97)).padStart(2, '0');
+
+export function isValidRijksregisternummer(nrn) {
+  const { birthDate, serial } = parse(nrn);
+  if (!(typeof nrn === 'string' && nrn.trim().length > 0)) return false;
+  if (nrn.length !== 11) {
+    return false;
+  }
+  if (isBisNumber(nrn)) return false;
+  const preNillies =
+    parseInt(nrn.slice(9, 11)) ===
```

Comment:
The parse util also returns the checksum, so we can reuse that here I think
```suggestion
    parseInt(checksum) ===
```

And then retrieve that from the parse util:
`const { birthDate, serial, checksum } = parse(nrn);`

Reply:
Indeed :+1: